main.cpp

```diff
@@ -49,13 +62,28 @@ void update(int value) {
     planeTFX += planeSpeed;
 
     glutPostRedisplay();
-    glutTimerFunc(10, update, 0);
+    glutTimerFunc(100, update, 0);
```

Comment:
But, this slow downs plane rendering, right?

Reply:
I tried to use multiple update function to see if it will work or not
But as i asked sir,he said as im using random func to generate particles the values will change